src/Writers/LaTeXWriter.jl

```diff
@@ -188,6 +189,8 @@ end
 function writeheader(io::IO, doc::Documents.Document)
     custom = joinpath(doc.user.root, doc.user.source, "assets", "custom.sty")
     isfile(custom) ? cp(custom, "custom.sty"; force = true) : touch("custom.sty")
+
+    tag = haskey(ENV, "CI") ? read_ci_env(false)[:tag] : ""
```

Comment:
You pass `false` here, but not above?

Not sure if the `finaldeploy` check should really be in the `read_ci_env` function like this in the first place.